File: leetcode/leetcode-python-solutions/implementstrstr.py
def str_str(haystack, needle):
    if needle == "":
        return 0
    for i in range(0, len(haystack)-len(needle)+1):
        for j in range(0, len(needle)):
            if needle[j] != haystack[i+j]:
                break
            if j == len(needle) - 1:
                return i
    return -1

File: leetcode/leetcode-js-solutions/implement-strStr/implementstrStr.js
// This is kind of cheating because you're using indexOf. 

const strStr = (haystack, needle) => {
  if (needle == "") {
    return 0; 
  }
  if (haystack.indexOf(needle) != -1) {
    return haystack.indexOf(needle); 
  } else {
    return -1; 
  }
} 


strStr("hello", "ll");
